myjsonrpc.pm

```diff
@@ -120,7 +120,7 @@ sub read_json {
         }
 
         my $qbuffer;
-        my $bytes = sysread($socket, $qbuffer, 8000);
+        my $bytes = sysread($socket, $qbuffer, 8000000);
```

Comment:
I would use a perl constant here. Maybe even modifiable via an environment variable, so testing different sizes is easier.

Reply:
done